pkgs/tools/typesetting/tex/texlive/bin.nix

```
@@ -38,7 +38,13 @@ let
       # http://mirrors.ctan.org/systems/doc/kpathsea/kpathsea.pdf for more
       # details
       sed -i '/^#define ST_NLINK_TRICK/d' texk/kpathsea/config.h
-    '';
+    '' +
+    # when cross compiling, we must use himktables from PATH
+    # (i.e. from buildPackages.texlive.bin.core.dev)
+    lib.optionalString (!(stdenv.buildPlatform.canExecute stdenv.hostPlatform))  ''
```

```suggestion
    lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```
Just FMI: Doesn't this also work?

---

I think so.  I can never remember the precedence rules for `!` and `&&`.  I guess it's like my own personal version of dyslexia.  I confess that it is at least part of why I opened https://github.com/NixOS/nixpkgs/pull/220435

---

https://github.com/NixOS/nixpkgs/pull/225035